feat: Improvements (#85)
solidity-labs-io/forge-proposal-simulator: docs/guides/introduction.md

### Step 3: Addresses File

> Create a JSON file following the standard on [Addresses](../overview/architecture/addresses.md). It is recommended to keep the JSON file in a separate folder, for example, `./addresses/31337.json`. Also allow read access for this file in `foundry.toml`.

Author: add that the file name should be same as chain id

> Create a JSON file following the standard on [Addresses](../overview/architecture/addresses.md). It is recommended to keep the JSON file in a separate folder, for example, `./addresses/31337.json`. The name of the JSON file would be the same as the network id. If there are multiple networks, addresses should be added in the JSON files corresponding to their network. Also, allow read access for this file in `foundry.toml`.

Reviewer: ```suggestion
Create a JSON file following the standard on [Addresses](../overview/architecture/addresses.md). It is recommended to keep the JSON file in a separate folder, for example, `./addresses/31337.json`. The name of the JSON file should be the same as the network id. If there are multiple networks, addresses should be added in the JSON files corresponding to their network. Be sure to allow read access for this file in `foundry.toml`.
```
